js/ui/calendar/ui.calendar.js

```diff
@@ -21,6 +21,9 @@ import messageLocalization from '../../localization/message';
 import dateLocalization from '../../localization/date';
 import { FunctionTemplate } from '../../core/templates/function_template';
 import { isCommandKeyPressed } from '../../events/utils/index';
+import CalendarSingleSelectionStrategy from './ui.calendar.single.selection.strategy';
```

Comment:
Why you decided that selection strategies should be part of calendar, not calendar.base part?

Reply:
Well we do not have calendar.base component. There is only base view, but as far as stratigy in needed to deside what options we should pass to view, I do not understand how strategies can be part of view

Reply:
Discussed this Anton, we agree that as far as strategies define how to render ranges and values, it's better to place them on View's level. But we have some functions (like for instance setting aria attributes) on ui.calendar level that also depends on strategies. So, moving strategies to view need more calendar refactoring and now we don't have time for this